requirements.yml

```diff
@@ -0,0 +1,10 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# roles/build-release is dependent on community.general collection
+collections:
+  - name: https://github.com/ansible-collections/community.general.git
+    type: git
+    version: main
```

Comment:
```suggestion
  - name: community.general
```

Reply:
If we use `type: git` and the c.g repo url,  then we can use it in the ci jobs as well.

What do you think? I am okay with this change as well.

Reply:
The git version is only good for CI testing, not for anything else. We should use the latest released version in the requirements file. (Which also means we cannot use the file for CI.)

Reply:
Cool! Committed the suggested changes:)